src/RecipeBot.Domain/Repositories/Data/RecipeEntryData.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (C) 2022 Dennis Tang. All rights reserved.
+//
+// This file is part of RecipeBot.
+//
+// RecipeBot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+namespace RecipeBot.Domain.Repositories.Data;
+
+/// <summary>
+/// Class representing a simplified recipe data entry.
```

Comment:
Not simplified, it's just a data object representing a recipe entry. Perhaps rename class as well

Reply:
No, the "Data" suffix is used to annotate classes that are not models being saved in the database and serve as a DTO to go through different layers (in this case from Persistency to Integration). 

Domain in this case is describing necessary functionality when needed